x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_list/index.tsx

```diff
@@ -153,6 +163,51 @@ export function getAgentsColumns({
         />
       ),
     },
+    ...(latestAgentVersionEnabled
+      ? [
+          {
+            field: AgentExplorerFieldName.AgentLastVersion,
+            name: (
+              <EuiToolTip
+                content={i18n.translate(
+                  'xpack.apm.agentExplorerTable.agentLatestVersionColumnTooltip',
+                  {
+                    defaultMessage: 'The latest released version of the agent.',
+                  }
+                )}
+              >
+                <>
+                  {i18n.translate(
+                    'xpack.apm.agentExplorerTable.agentLatestVersionColumnLabel',
+                    { defaultMessage: 'Latest Agent Version' }
+                  )}
+                  &nbsp;
+                  <EuiIcon
+                    size="s"
+                    color="subdued"
+                    type="questionInCircle"
+                    className="eui-alignCenter"
+                  />
+                </>
+              </EuiToolTip>
+            ),
+            width: '10%',
+            align: 'center',
+            truncateText: true,
+            render: (
+              _: any,
```

Comment:
why `any` here? I understand it's not being used but I don't see a reason to downgrade it to `any`. Do you need the types in the function at all?

Reply:
I'm having the following problem if I don't declare the `AgentExplorerItem` type
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/1313018/230393806-e5cbc92a-c0b5-4d04-b9c4-63ceaf1e9d38.png">
and then if I declare the `AgentExplorerType` but don't declare the type of value(`_`) I get this
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/1313018/230394102-28e8842e-6554-4931-96c7-7e2c7da18f64.png">